pkg/usage/metrics.go

```diff
@@ -0,0 +1,56 @@
+package usage
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type Metrics struct {
+	pageViewsTotal       *prometheus.CounterVec
+	pageImpressionsTotal *prometheus.CounterVec
+}
+
+func (m *Metrics) GetCollectors() []prometheus.Collector {
+	return []prometheus.Collector{
+		m.pageViewsTotal,
+		m.pageImpressionsTotal,
+	}
+}
+
+func (m *Metrics) HandlePageView(perspective string) {
```

Comment:
We should probably return error here and below. It will be also good noticing any failures